### DISCUSSION TITLE
file attachment by DnD

### DISCUSSION THREAD

**Discussion by bmm-alc** [outline/outline#3360] [Ideas]
Hi there,

Thanks for doing outline, it's is simple to run and really effective.
I've upgraded the docker version to `0.63.0-1` to try file attachment, it's working fine, it would be better if that could be done by just doing a drag and drop from file manager application.
I've also notice this is possible to only attach one file at the time, could it be possible to attach several files at once.

Best

**Comment by tommoor** [comment]
Both of these things should work on that release and were part of the original feature set… perhaps you can upload a video of the problem.

**Reply by bmm-alc** [reply]
Hello

https://user-images.githubusercontent.com/80455946/162894445-8266e35e-443f-4f15-93bc-a2cb203bb516.mp4
When I do a dnd wether tere is one file or several it does not work (I use firefox on Linux)

https://user-images.githubusercontent.com/80455946/162894451-be350a8c-73a1-4552-8944-f94be92183ca.mp4
when I use the `/file` command I cannot select several files in the file selector.

Hopes it helps

Best.

**Reply by tommoor** [reply]
This is useful and is just confusing UX. When you have an empty document like this you must drag onto the empty line at the top – not the white space below which technically isn't the "editor". We'll have to make this work somehow.
